src/helpers/geometry.rs

```diff
@@ -0,0 +1,22 @@
+use crate::{TilemapSize, TilemapTileSize, Transform};
+use bevy::math::Vec2;
+
+/// Calculates a [`Vec2`] position for a tilemap so that when set to this position, it shows up
+/// centered on the screen.
+pub fn get_tilemap_center(size: &TilemapSize, tile_size: &TilemapTileSize) -> Vec2 {
```

Comment:
If our tiles are centered now do we need this? Or is that only the case for hex/iso?

Reply:
I am not sure, but I think so? This is for centering the whole tilemap, right? Currently, our tiles are "centered" in the sense that the center of a tile with `TilePos(0, 0)` lies at `[0, 0]` in world space.